SudokuSolver/SudokuSolver/SudokuSolver.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Extensions;
+
+namespace SudokuSolver
+{
+    public class SudokuSolver
+    {
+        public class EmptySudokuCube
+        {
+            public int Row { get; init; }
+            public int Col { get; init; }
+
+            public List<int> PossibleValues { get; init; }
+            public EmptySudokuCube(int row, int col)
+            {
+                Row = row;
+                Col = col;
+                PossibleValues = new List<int>();
+            }
+        }
+
+        private int[,] _grid;
+        private int _maxRows;
+        private int _maxCols;
```

Comment:
maxRows chyba nie jest trafioną nazwą. Nie jest to maksymalna liczba wierszy tylko po prostu liczba wierszy. Proponowałbym zmienić na "_rowsNumber" na przykład